packages/components/tabs/src/tab-group.ts

```diff
@@ -414,7 +418,7 @@ export class TabGroup extends ScopedElementsMixin(LitElement) {
       this.menuItems = undefined;
     }
 
-    this.selectedTab?.scrollIntoView();
+    // this.selectedTab?.scrollIntoView(); // TODO: causes unnecessary scrolling down on tab change when there is a sticky element
```

Comment:
Is this issue reproducible somewhere? I'd rather remove the code entirely if that fixes the issue than leave it in here.

Reply:
When you'll uncomment it and try to run the website or even story in storybook with sticky tabs, you can see that it's scrolling down or up only to the sticky element, not totally to the top. Should I remove it then?